libnavigation-resource/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 29
```

Comment:
We should use `ext` values defined in `[dependencies.gradle](https://github.com/mapbox/mapbox-navigation-android/blob/master/gradle/dependencies.gradle). Same thing applies to `dependenciesList` 👇